structures/MisakiClient.js

```diff
@@ -0,0 +1,163 @@
+const { Client, Collection, MessageEmbed } = require("discord.js");
+const CommandStore = require(`${process.cwd()}/structures/CommandStore.js`);
+const EventStore = require(`${process.cwd()}/structures/EventStore.js`);
+const Enmap = require("enmap");
+const EnmapLevel = require("enmap-level");
+const klaw = require("klaw");
+const path = require("path");
+const idioticApi = require("idiotic-api");
+const moment = require("moment");
+require("moment-duration-format");
+
+class MisakiClient extends Client {
+  constructor(options) {
+    super(options);
+
+    this.config = require(`${process.cwd()}/config.js`);
+    this.logger = require(`${process.cwd()}/util/Logger`);
+    this.responses = require(`${process.cwd()}/assets/responses.js`);
+    this.idiotAPI = new idioticApi.Client(this.config.apiTokens.idiotToken, { dev: true });
```

Comment:
i would recommend desctructuring here

Reply:
with how the api wrapper is written, de-structuring is basically impossible

Reply:
i dont say you should destructure the methods out of it, i just wanted that you descructure the Client out of the package, you can also rename `Client` to `IdioticClient` or something simular since the `Client` name is alrweady taken in that scope

Reply:
well actually you can ignore that its kinda pointless since you would need to rename it